### ISSUE TITLE
small issues observed when using `ez_zarr`

### ISSUE THREAD

**Issue by mbstadler** [fmicompbio/ez_zarr#16]
I am adding these issues communicated to me to not forget them and to keep track of what we have fixed:

I was working with ez_zarr today ☺️ and I noticed a couple of things: 
- [ ] the names (from get_names()) after import_fractal_plate() are saying B2 instead of B02.
  *This seems to already be fixed (at least I cannot reproduce it with v0.3.1-devel)*
- [x] the documentation is synced to the GitHub repo but not to the pypi release.. so for users (not users/developers) the documentation might be unhelpful..or can you decide which documentation to browse? E.g. select the pypi release? Our documentation is currently not versioned and always refers to the main branch GitHub.
  *I think the asynchrony is usually not an issue if we keep the development cycles short, and especially don't wait long between pushing to GitHub and creating a new release. I will try to do this in the future and it should further get better as changes become less drastic.*
- [x] if we need to test the new functionalities, I think it's useful we make the releases. Oftentimes I am scripting for other users, so I want to use the pypi release, and today for instance I was solving something for someone else and having the get_label_names() propagating for all wells in an ImageList, as well as plotting with list of segmentations would have been useful!
  *I fully agree - I have released v0.3.0 already and as mentioned will try to minimize the time between GitHub updates and releases.*

**Comment by mbstadler** [comment]
I think these are all addressed (if only in working practise and not in code ;-). Feel free to reopen this issue if you see anything that is still open.

**Comment by csoneson** [comment]
Regarding the following: 

- the names (from get_names()) after import_fractal_plate() are saying B2 instead of B02.
*This seems to already be fixed (at least I cannot reproduce it with v0.3.1-devel)*

Won't that depend on how many columns the plate is inferred to have? From https://github.com/fmicompbio/ez_zarr/blob/21cb813397bc60d147e5eb1e0a74fa96ea1d3341/src/ez_zarr/ome_zarr.py#L139-L141 the names will be `B2` etc if the inferred plate layout has less than 10 columns (which could of course be changed).

**Comment by mbstadler** [comment]
Good point - I am reopening the issue, the names should be always `B03` even for a plate with less than 10 columns.

**Comment by csoneson** [comment]
PR with suggested fix: https://github.com/fmicompbio/ez_zarr/pull/17 (removing the `create_name_plate_A1()` function, as we did with `create_name_plate_A001()`)

**Comment by imagejan** [comment]
According to the [NGFF spec (plate metadata)](https://ngff.openmicroscopy.org/latest/index.html#plate-md), the names for columns and rows are listed in `columns` and `rows` fields of the json metadata, and `wells` is linking to the respective indices of `columns` and `rows`. (I don't think it's a good idea to hard-code a convention here.)

**Comment by mbstadler** [comment]
Thanks @imagejan for pointing this out, we'll read it from there.

**Comment by mbstadler** [comment]
Has been addressed in #17
